File: app/src/main/java/com/whitecoats/adapter/QuickActionItemHolder.java
package com.whitecoats.adapter;

import androidx.recyclerview.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.whitecoats.clinicplus.R;

public class QuickActionItemHolder extends RecyclerView.ViewHolder {

    private TextView carTitleText = null;

    private ImageView carImageView = null;

    public QuickActionItemHolder(View itemView) {
        super(itemView);

        if(itemView != null)
        {
            carTitleText = (TextView)itemView.findViewById(R.id.card_view_image_title);

            carImageView = (ImageView)itemView.findViewById(R.id.card_view_image);
        }
    }

    public TextView getCarTitleText() {
        return carTitleText;
    }

    public ImageView getCarImageView() {
        return carImageView;
    }
}

File: app/src/main/java/com/whitecoats/clinicplus/adapters/QuickActionItemHolder.kt
package com.whitecoats.clinicplus.adapters

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.whitecoats.clinicplus.R

class QuickActionItemHolder(itemView: View?) : RecyclerView.ViewHolder(
    itemView!!
) {
    lateinit var carTitleText: TextView
    lateinit var carImageView: ImageView

    init {
        if (itemView != null) {
            carTitleText = itemView.findViewById<View>(R.id.card_view_image_title) as TextView
            carImageView = itemView.findViewById<View>(R.id.card_view_image) as ImageView
        }
    }
}